ActivityListener.Tests/E2ETests/Stories/ProcessActivityEventTests.cs

```diff
@@ -60,15 +81,27 @@ public void UnknownEventTypeThrows()
                 .BDDfy();
         }
 
+
         [Theory]
-        [MemberData(nameof(AllEventTypes))]
-        public void EventCreatesActivityHistoryRecord(string eventType)
+        [MemberData(nameof(AllActionableEventTypes))]
+        public void EventCreatesActivityHistoryRecordForAnActionableEvent(string eventType)
         {
+
             this.Given(g => _steps.GivenAnEntityActivityEvent(eventType))
                 .When(w => _steps.WhenTheFunctionIsTriggered(_steps.EventSns))
                 .Then(t => _steps.ThenAnActivityHistoryRecordIsCreatedAsync(_dbFixture.DynamoDbContext, _steps.EventSns))
                 .BDDfy();
         }
 
+        [Theory]
+        [MemberData(nameof(NotActionableEventTypes))]
+        public void EventDoesNotCreateActivityHistoryRecordForANotActionableEvent(string eventType)
+        {
+            this.Given(g => _steps.GivenAnEntityActivityEvent(eventType))
+                .When(w => _steps.WhenTheFunctionIsTriggered(_steps.EventSns))
+                .Then(t => _steps.ThenNoActivityHistoryRecordIsCreatedAsync(_dbFixture.DynamoDbContext, _steps.EventSns))
+                .BDDfy();
+        }
+
```

Comment:
These test updates verify that all events except the new ones that weren't handled so far will be processed